t/stats.t

```diff
@@ -203,6 +205,15 @@ is(0, $stats->{'evictions'});
 is(0, $stats->{'reclaimed'});
 is(0, $stats->{'lrutail_reflocked'});
 
+# item expired
+print $sock "set should_expire 0 1 6\r\nfooval\r\n";
+is(scalar <$sock>, "STORED\r\n", "set item to expire");
+sleep(2);
```

Comment:
you can set an expiration time of 31 days. Past 30 days it's interpreted as a unix timestamp, so any still relatively-small value will be immediately expired.